Makefile

```diff
@@ -98,6 +98,10 @@ docker-system-tests: docker-compose.override.yml
 docker-compose.override.yml:
 	printf "version: '2.3'\nservices:\n beat:\n  build:\n   args: [UID=%d]" $(shell id -u) > $@
 
+.PHONY: test-packages-install
```

Comment:
This is such a specific thing to have in the Makefile. How would you feel about reverting this, and changing `script/jenkins/test-install-packages.sh` as suggested below?